suites/apis/requestorTest.js

```diff
@@ -0,0 +1,84 @@
+  home, login, users,
+}) => {
+  home.do.goToHomepage();
+  login.complete.login(users.user0);
+  const response = await requestorTasks.createRequestFromPolicyID(users.user0, 'random-policy');
+  expect(response).to.have.property('status_code', 400);
+});
+
+Scenario('User does not have access request for a policy and requests for access followed by sending a Revoke request @requestor', async ({
+  home, login, users, fence,
+}) => {
+  home.do.goToHomepage();
+  login.complete.login(users.user0);
+
+  // Check that the user does not have access to policy 'requestor_integration_test'.
+  let userInfo = await fence.do.getUserInfo(users.user0.accessToken);
+  expect(userInfo.data.authz).to.have.property('/requestor-integration-test');
+  expect(userInfo.data.authz['/requestor-integration-test']).to.deep.to.not.include({ method: 'read', service: 'integration' });
+
+  // Create a request for a policy that is present in Arborist
+  const createResponse = await requestorTasks.createRequestFromPolicyID(users.user0, 'requestor_integration_test');
+  expect(createResponse).to.have.property('status_code', 201);
+
+  // Check if the access is created -- It shouldn't since it is not signed yet
+  userInfo = await fence.do.getUserInfo(users.user0.accessToken);
+  expect(userInfo.data.authz['/requestor-integration-test']).to.deep.to.not.include({ method: 'read', service: 'integration' });
+
+  // Create a PUT request to sign the previous request using an authenticated user
+  await requestorTasks.signedRequest(createResponse.request_id);
+
+  // Verify if access is given to the user -- There should be
+  userInfo = await fence.do.getUserInfo(users.user0.accessToken);
+  expect(userInfo.data.authz['/requestor-integration-test']).to.deep.to.include({ method: 'read', service: 'integration' });
+
+  // Create a request to 'revoke' a policy that is present in Arborist
+  const revokeResponse = await requestorTasks.createRequestFromPolicyID(users.user0, 'requestor_integration_test', true);
+
+  // Check if the access is revoked -- It shouldn't since it is not signed yet
+  userInfo = await fence.do.getUserInfo(users.user0.accessToken);
+  expect(userInfo.data.authz['/requestor-integration-test']).to.deep.to.include({ method: 'read', service: 'integration' });
+
+  // Create a PUT request to sign the previous request using an authenticated user
+  await requestorTasks.signedRequest(revokeResponse.request_id);
+
+  // Verify if access is revoked from the user
+  userInfo = await fence.do.getUserInfo(users.user0.accessToken);
+  expect(userInfo.data.authz['/requestor-integration-test']).to.deep.to.not.include({ method: 'read', service: 'integration' });
+});
+
+// Create a revoke request for a policy that exists in Arborist
+// but user does not have access to=> 400 error
+Scenario('User sends a revoke request to a property he does not have access to @requestor', async ({
```

Comment:
Also based on my previous comment - the user sending the request would not be the user who does not have the policy
```suggestion
Scenario('Send a revoke request for a policy the user does not have @requestor', async ({
```